proto/cacheclient.proto

```diff
@@ -54,33 +54,36 @@ message _SetResponse {
 
 message _DictionaryGetRequest {
   bytes dictionary_name = 1;
-  bytes dictionary_key = 2;
+  repeated bytes dictionary_keys = 2;
 }
 
 message _DictionaryGetResponse {
-  ECacheResult result = 1;
-  bytes cache_body = 2;
-  string message = 3;
+  message _DictionaryGetResponsePart {
+    ECacheResult result = 1;
+    bytes cache_body = 2;
+  }
+  repeated _DictionaryGetResponsePart dictionary_body = 1;
```

Comment:
just out of curiosity, was this a customer ask?  or does this exist in other cache engine APIs?  can follow up offline

Reply:
I followed up offline with a link to the conversation

Reply:
thanks!